src/routes/Dashboard/components/Dashboard.scss

```diff
@@ -5,9 +5,18 @@
 .tabs {
   width: 100%;
   height: 600px;
+  overflow: scroll;
 }
 
 .map {
   width: 100%;
   height: 600px;
 }
+
+pre {
```

Comment:
Yup. I'm cleaning things up for a v1 cut and demo Monday. We will make tasks for those for next sprint.